### DISCUSSION TITLE
Halmet "check engine" indicator in N2K with Raymarine Axiom

### DISCUSSION THREAD

**Discussion by pmitham** [hatlabs/discussions#118] [General]
Finally had a chance recently to add the Halmet to my N2K network (no sensors connected yet)  on my Axiom I actually see a "check engine" light!  what code would be sending this information?

**Comment by mairas** [comment]
I dunno, maybe there's something wrong with your engine?

Joking aside - if you're using the HALMET example firmware, in the `main.cpp` file around line 217 onwards you have some digital alarm inputs defined. If you haven't disabled them, they're the culprit.

**Reply by pmitham** [reply]
I had a look and yes the default didigtal inputs are still active,  but interesting that it creates a "check engine" light! I'll shut them off for now
